src/Database/Expression/CaseExpression.php

```diff
@@ -52,15 +59,28 @@ class CaseExpression implements ExpressionInterface
     /**
      * Constructs the case expression
      *
-     * @param array|\Cake\Database\ExpressionInterface $conditions The conditions to test. Must be a ExpressionInterface
+     * @param array|mixed|\Cake\Database\ExpressionInterface $conditions The conditions to test. Must be a ExpressionInterface
```

Comment:
Why mixed? That kind of allows everything now.

Reply:
Sorry, fixed and updated